app/views/sleeplogs/index.html.haml

```diff
@@ -2,8 +2,16 @@
   .text-center.sleeplogs_menu.small-2.small-offset-1.columns#graph_link= link_to "GRAPH", "#"
   .text-center.sleeplogs_menu.small-2.small-offset-1.columns#log_link= link_to "LOG", "#"
   .small-1.columns
+%br
 .row
-  .small-11.small-offset-1.columns#add_sleeplog_link= link_to "ADD SLEEPLOG", new_sleeplog_path, remote: true
+  %a.small-offset-1.columns{"data-reveal-id" => "myModal", :href => "#"} ADD A NEW SLEEPLOG
```

Comment:
Line is too long. [91/80]<br>Hash attribute should start with one space after the opening brace<br>Hash attribute should end with one space before the closing brace